Library/Formula/h264enc.rb

```diff
@@ -0,0 +1,76 @@
+class H264enc < Formula
+  homepage "http://h264enc.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/h264enc/h264enc/h264enc-9.9.0.tar.gz"
+  sha1 "8bfa7017386a4b7612acd21e48dedf940652582d"
+
+  # depends_on "cmake" => :build
+  # depends_on :x11 # if your formula requires any X11/XQuartz components
+  # [encode, x264], mplayer needs x264 installed first 
```

Comment:
The mplayer formula doesn't specify these options. If the x264 library is present, mplayer links it in.  I have a crude test to see if it is built correctly. I also don't know how to specify versions (gpac >= 0.4.5, which seems to be the case). 

Cleaned up comment lines and pushed changes.